examples/getstarted/config/features.js

```diff
@@ -0,0 +1,5 @@
+module.exports = ({ env }) => ({
+  future: {
+    contentReleases: env.bool('STRAPI_FEATURES_FUTURE_CONTENTRELEASES', false),
```

Comment:
Camelcase conversion to pascal casse will split different words. it's important to do the same here.

```suggestion
    contentReleases: env.bool('STRAPI_FEATURES_FUTURE_CONTENT_RELEASES', false),
```